notifier/events/event.go

```diff
@@ -43,7 +43,15 @@ func (worker *FetchEventsWorker) Start() {
 						}
 						continue
 					}
+
 					worker.Metrics.EventsReceived.Mark(1)
+					stateMeterName := event.OldState.String() + "_to_" + event.State.String()
```

Comment:
What do you think about states as a nodes of metric? Like `OK.to.ERROR` instead of `OK_to_ERROR`.

Reply:
I thought of this, but it sounds inconvenient. I can't imagine a scenario in which it would be necessary.